Project does not build on Windows due to NLS characters in code / no source file encoding specified

The project does not build on Windows, due to the use of NLS chars in source code. Unless the javac option "-encoding" is specified, the platform default converter is used (http://docs.oracle.com/javase/6/docs/technotes/tools/windows/javac.html).

(Source code based on revision 49798f05eabbac0c71f486a450abeda0e497d6b4)
```
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:testCompile (default-testCompile) on project xmltordf: Compilation failure: Compilation failure:
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[681,23] illegal character: '\u00b8'
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[682,23] illegal character: '\u00a6'
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[689,52] illegal character: '\u00b8'
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[689,77] illegal character: '\u00a6'
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[689,78] illegal start of expression
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[689,79] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[690,13] illegal start of expression
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[690,31] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[690,38] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[690,49] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[690,52] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[690,55] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[690,84] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[690,87] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[691,21] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[696,52] illegal character: '\u00b8'
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[696,77] illegal character: '\u00a6'
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[696,78] illegal start of expression
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[696,79] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[697,13] illegal start of expression
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[697,31] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[697,38] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[697,49] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[697,52] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[697,55] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[697,84] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[697,87] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[698,21] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[703,52] illegal character: '\u00b8'
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[703,77] illegal character: '\u00a6'
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[703,78] illegal start of expression
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[703,79] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[704,13] illegal start of expression
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[704,31] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[704,38] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[704,49] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[704,52] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[704,55] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[704,84] not a statement
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[704,87] ';' expected
[ERROR] /C:/git/XmlToRdf/xmltordf/src/test/java/XmlToRdfTest.java:[705,21] ';' expected
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

Yes, with that branch it compiles nicely :-).

However, it fails while executing tests:

```
Tests run: 64, Failures: 0, Errors: 1, Skipped: 1, Time elapsed: 0.999 sec <<< FAILURE!
specialShallowHandling(XmlToRdfTest)  Time elapsed: 0.006 sec  <<< ERROR!
org.apache.jena.atlas.RuntimeIOException: java.nio.charset.MalformedInputException: Input length = 1
        at org.apache.jena.atlas.io.IO.exception(IO.java:216)
        at org.apache.jena.atlas.io.CharStreamBuffered$SourceReader.fill(CharStreamBuffered.java:77)
        at org.apache.jena.atlas.io.CharStreamBuffered.fillArray(CharStreamBuffered.java:154)
        at org.apache.jena.atlas.io.CharStreamBuffered.advance(CharStreamBuffered.java:137)
        at org.apache.jena.atlas.io.PeekReader.advanceAndSet(PeekReader.java:235)
        at org.apache.jena.atlas.io.PeekReader.init(PeekReader.java:229)
        at org.apache.jena.atlas.io.PeekReader.peekChar(PeekReader.java:151)
        at org.apache.jena.atlas.io.PeekReader.makeUTF8(PeekReader.java:92)
        at org.apache.jena.riot.tokens.TokenizerFactory.makeTokenizerUTF8(TokenizerFactory.java:48)
        at org.apache.jena.riot.lang.RiotParsers.createParser(RiotParsers.java:61)
        at org.apache.jena.riot.RDFParserRegistry$ReaderRIOTLang.read(RDFParserRegistry.java:171)
        at org.apache.jena.riot.RDFDataMgr.process(RDFDataMgr.java:861)
        at org.apache.jena.riot.RDFDataMgr.parse(RDFDataMgr.java:667)
        at org.apache.jena.riot.RDFDataMgr.read(RDFDataMgr.java:212)
        at org.apache.jena.riot.RDFDataMgr.read(RDFDataMgr.java:145)
        at org.apache.jena.riot.RDFDataMgr.read(RDFDataMgr.java:134)
        at org.apache.jena.riot.adapters.AdapterFileManager.readModelWorker(AdapterFileManager.java:282)
        at org.apache.jena.util.FileManager.readModel(FileManager.java:341)
        at org.apache.jena.util.FileManager.readModel(FileManager.java:325)
        at XmlToRdfTest.testAdvancedStream(XmlToRdfTest.java:1333)
        at XmlToRdfTest.specialShallowHandling(XmlToRdfTest.java:685)
        ... (stacktrace from maven and junit ommitted)
Caused by: java.nio.charset.MalformedInputException: Input length = 1
        at java.nio.charset.CoderResult.throwException(CoderResult.java:281)
        at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:339)
        at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
        at java.io.InputStreamReader.read(InputStreamReader.java:184)
        at java.io.Reader.read(Reader.java:140)
        ... 51 more
```

This is probably a different problem, though, caused by an encoding problem in a test file.

--- a/README.md
+++ b/README.md
@@ -46,7 +46,7 @@ To use XmlToRdf in your project add the following dependency to your pom.xml fil
 <dependency>
     <groupId>no.acando</groupId>
     <artifactId>xmltordf</artifactId>
-    <version>1.4.13</version>
+    <version>1.4.14</version>
 </dependency>
 ```
 
@@ -57,7 +57,7 @@ Two steps are required for this. First you need to install the jar file in your 
 ```
  mvn \
     install:install-file \
-    -Dfile=xmltordf/target/xmltordf-1.4.13.jar \
+    -Dfile=xmltordf/target/xmltordf-1.4.14.jar \
     -DpomFile=xmltordf/pom.xml \
     -DlocalRepositoryPath=/INSTALL_DIRECTORY
 
@@ -600,14 +600,14 @@ Builder.getAdvancedBuilderStream()
         ex:nr     "0000002" ;
         ex:title  "Other record" .
 
-[ a                  ex:archive ;
-  xmlToRdf:hasChild  <http://acme.com/records/0000002> , <http://acme.com/records/0000001>
-] .
-
 <http://acme.com/records/0000001>
         a         ex:record ;
         ex:nr     "0000001" ;
         ex:title  "Important record" .
+
+[ a                  ex:archive ;
+  xmlToRdf:hasChild  <http://acme.com/records/0000002> , <http://acme.com/records/0000001>
+] .
 
 ```
 
@@ -1418,19 +1418,19 @@ Builder.getAdvancedBuilderStream()
 @prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
 
 _:b0    a                  ex:b ;
+        xmlToRdf:hasValue  "World" .
+
+_:b1    a                  ex:b ;
         xmlToRdf:hasValue  "Hello" .
 
 [ a                  ex:document ;
   xmlToRdf:hasChild  [ a                         ex:paragraph ;
-                       xmlToRdf:hasChild         _:b0 , _:b1 ;
-                       xmlToRdf:hasMixedContent  ( _:b0 _:b1 " !" ) ;
+                       xmlToRdf:hasChild         _:b1 , _:b0 ;
+                       xmlToRdf:hasMixedContent  ( _:b1 _:b0 " !" ) ;
                        xmlToRdf:hasValue         "!"
                      ] ;
   ex:paragraph       "Hello, World!"
 ] .
-
-_:b1    a                  ex:b ;
-        xmlToRdf:hasValue  "World" .
 
 ```
 
@@ -1549,7 +1549,7 @@ Builder.getAdvancedBuilderStream()
 @prefix ex:    <http://example.org/> .
 @prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
 
-ex:ea469e53-2e64-4757-94cf-6ff60c6c71ce
+ex:560ec258-b775-4406-823e-e93df5578ab7
         a        ex:people ;
         ex:name  "John Doe" .
 

--- a/doclet/src/main/java/no/acando/xmltordf/doclet/JsonJavadocExampleRunner.java
+++ b/doclet/src/main/java/no/acando/xmltordf/doclet/JsonJavadocExampleRunner.java
@@ -38,7 +38,6 @@ import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.parsers.ParserConfigurationException;
 import java.io.*;
 import java.lang.reflect.Type;
-import java.nio.charset.Charset;
 import java.util.ArrayList;
 import java.util.List;
 
@@ -49,6 +48,7 @@ public class JsonJavadocExampleRunner {
         "import no.acando.xmltordf.doclet.ExampleInterface;\n" +
         "import org.xml.sax.SAXException;\n" +
         "import java.io.*;\n" +
+        "import java.nio.charset.Charset;\n"+
         "import no.acando.xmltordf.Builder;\n" +
         "import javax.xml.parsers.ParserConfigurationException;\n" +
         "import java.io.ByteArrayInputStream;\n" +
@@ -63,7 +63,7 @@ public class JsonJavadocExampleRunner {
         " ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();\n" +
         "\t\t\ttry {\n" +
         "\t\t\t\tBUILDER.convertToStream(new ByteArrayInputStream(xml.getBytes(\"UTF-8\")), byteArrayOutputStream);\n" +
-        "\t\t\t\tString s = new String(byteArrayOutputStream.toByteArray(), Charset.forName(\"UTF-8\");\n" +
+        "\t\t\t\tString s = new String(byteArrayOutputStream.toByteArray(), Charset.forName(\"UTF-8\"));\n" +
         "\t\t\t\treturn s;\n" +
         "\t\t\t} catch (Exception e) {\n" +
         "\t\t\t\te.printStackTrace();\n" +

--- a/doclet/src/main/java/no/acando/xmltordf/doclet/JsonJavadocExampleRunner.java
+++ b/doclet/src/main/java/no/acando/xmltordf/doclet/JsonJavadocExampleRunner.java
@@ -38,6 +38,7 @@ import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.parsers.ParserConfigurationException;
 import java.io.*;
 import java.lang.reflect.Type;
+import java.nio.charset.Charset;
 import java.util.ArrayList;
 import java.util.List;
 
@@ -61,8 +62,8 @@ public class JsonJavadocExampleRunner {
         "\tpublic String toString(String xml) {\n" +
         " ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();\n" +
         "\t\t\ttry {\n" +
-        "\t\t\t\tBUILDER.convertToStream(new ByteArrayInputStream(xml.getBytes()), byteArrayOutputStream);\n" +
-        "\t\t\t\tString s = new String(byteArrayOutputStream.toByteArray());\n" +
+        "\t\t\t\tBUILDER.convertToStream(new ByteArrayInputStream(xml.getBytes(\"UTF-8\")), byteArrayOutputStream);\n" +
+        "\t\t\t\tString s = new String(byteArrayOutputStream.toByteArray(), Charset.forName(\"UTF-8\");\n" +
         "\t\t\t\treturn s;\n" +
         "\t\t\t} catch (Exception e) {\n" +
         "\t\t\t\te.printStackTrace();\n" +
@@ -121,7 +122,7 @@ public class JsonJavadocExampleRunner {
 
                         Model defaultModel = ModelFactory.createDefaultModel();
 
-                        defaultModel.read(new ByteArrayInputStream(s.getBytes()), "", "TTL");
+                        defaultModel.read(new ByteArrayInputStream(s.getBytes("UTF-8")), "", "TTL");
 
                         defaultModel.setNsPrefix("ex", "http://example.org/");
                         defaultModel.setNsPrefix("xmlToRdf", "http://acandonorway.github.com/XmlToRdf/ontology.ttl#");

--- a/examples/brreg/src/main/java/Main.java
+++ b/examples/brreg/src/main/java/Main.java
@@ -36,7 +36,7 @@ public class Main {
         Model naeringskode = ModelFactory.createDefaultModel();
 
         for (int i = 1; true; i++) {
-            ByteArrayInputStream naeringskodeXml = new ByteArrayInputStream(IOUtils.toString(new URI("https://hotell.difi.no/api/xml/brreg/naeringskode?page=" + i)).getBytes());
+            ByteArrayInputStream naeringskodeXml = new ByteArrayInputStream(IOUtils.toString(new URI("https://hotell.difi.no/api/xml/brreg/naeringskode?page=" + i)).getBytes("UTF-8"));
 
             Model naeringskodeTemp = Builder.getAdvancedBuilderJena()
                 .setBaseNamespace(ns, Builder.AppliesTo.bothElementsAndAttributes)
@@ -49,7 +49,7 @@ public class Main {
                 .mappedTo((elementMap, attributeMap) -> "http://brreg.no/naeringskode/"+elementMap.get("http://brreg.no/naerk"))
 
                 .build().convertForPostProcessing(naeringskodeXml)
-                .mustacheTransform(new ByteArrayInputStream("delete {?a ?b ?c} where {?a ?b ?c. filter(isBlank(?a))}".getBytes()), new Object())
+                .mustacheTransform(new ByteArrayInputStream("delete {?a ?b ?c} where {?a ?b ?c. filter(isBlank(?a))}".getBytes("UTF-8")), new Object())
                 .getModel();
 
             naeringskode.add(naeringskodeTemp);
@@ -61,7 +61,7 @@ public class Main {
         }
 
 
-        ByteArrayInputStream orgFormXml = new ByteArrayInputStream(IOUtils.toString(new URI("https://hotell.difi.no/api/xml/brreg/organisasjonsform?")).getBytes());
+        ByteArrayInputStream orgFormXml = new ByteArrayInputStream(IOUtils.toString(new URI("https://hotell.difi.no/api/xml/brreg/organisasjonsform?")).getBytes("UTF-8"));
 
         Model enhetstyper = Builder.getAdvancedBuilderJena()
             .setBaseNamespace(ns, Builder.AppliesTo.bothElementsAndAttributes)
@@ -75,11 +75,11 @@ public class Main {
             .mappedTo((elementMap, attributeMap) -> "http://brreg.no/enhetstype/"+elementMap.get("http://brreg.no/enhetstype"))
 
             .build().convertForPostProcessing(orgFormXml)
-            .mustacheTransform(new ByteArrayInputStream("delete {?a ?b ?c} where {?a ?b ?c. filter(isBlank(?a))}".getBytes()), new Object())
+            .mustacheTransform(new ByteArrayInputStream("delete {?a ?b ?c} where {?a ?b ?c. filter(isBlank(?a))}".getBytes("UTF-8")), new Object())
             .getModel();
 
 
-        ByteArrayInputStream brregXml = new ByteArrayInputStream(IOUtils.toString(new URI("https://hotell.difi.no/api/xml/brreg/enhetsregisteret?")).getBytes());
+        ByteArrayInputStream brregXml = new ByteArrayInputStream(IOUtils.toString(new URI("https://hotell.difi.no/api/xml/brreg/enhetsregisteret?")).getBytes("UTF-8"));
 
 
         ComplexClassTransform convertDate = e -> {

--- a/examples/reportingHub/src/main/java/Main.java
+++ b/examples/reportingHub/src/main/java/Main.java
@@ -34,7 +34,7 @@ public class Main {
 
         String url = "http://www.psa.no/getfile.php/PDF/DDRS_example.xml";
 
-        ByteArrayInputStream reportingHubXml = new ByteArrayInputStream(IOUtils.toString(new URI(url)).getBytes());
+        ByteArrayInputStream reportingHubXml = new ByteArrayInputStream(IOUtils.toString(new URI(url)).getBytes("UTF-8"));
 
         String witsml = "http://www.witsml.org/schemas/1series#";
 
@@ -147,7 +147,7 @@ public class Main {
                 "       ?uri a <http://data.posccaesar.org/npd/ontology/ilap-interface/Wellbore>; ",
                 "            rdfs:label ?wellboreString . ",
                 "   }",
-                "}").getBytes()),
+                "}").getBytes("UTF-8")),
 
                 new Object() {
                     String witsmlNs = witsml;
@@ -155,7 +155,7 @@ public class Main {
             .mustacheTransform(new ByteArrayInputStream(String.join("\n", "",
                 "delete{?a <http://acandonorway.github.com/XmlToRdf/ontology.ttl#hasValue> ?b}",
                 "insert{?a <{{{witsmlNs}}}value> ?b}",
-                "where{?a <http://acandonorway.github.com/XmlToRdf/ontology.ttl#hasValue> ?b}").getBytes()), new Object() {
+                "where{?a <http://acandonorway.github.com/XmlToRdf/ontology.ttl#hasValue> ?b}").getBytes("UTF-8")), new Object() {
                 String witsmlNs = witsml;
             })
             .getModel();

--- a/examples/yr/src/main/java/Main.java
+++ b/examples/yr/src/main/java/Main.java
@@ -37,7 +37,7 @@ public class Main {
 
 //        System.out.println(IOUtils.toString(new URI(url)));
 
-        ByteArrayInputStream yrXml = new ByteArrayInputStream(IOUtils.toString(new URI(url)).getBytes());
+        ByteArrayInputStream yrXml = new ByteArrayInputStream(IOUtils.toString(new URI(url)).getBytes("UTF-8"));
 
         Builder.AdvancedJena builder = Builder.getAdvancedBuilderJena()
             .convertComplexElementsWithOnlyAttributesAndSimpleTypeChildrenToPredicate(true)

--- a/pom.xml
+++ b/pom.xml
@@ -14,6 +14,10 @@
         <module>examples</module>
     </modules>
 
+    <properties>
+         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
     <dependencies>
 
 

--- a/xmltordf/pom.xml
+++ b/xmltordf/pom.xml
@@ -11,6 +11,11 @@
     <description>Fast and configurable XML to RDF converter.</description>
     <url>https://github.com/AcandoNorway/XmlToRdf</url>
 
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
     <licenses>
         <license>
             <name>Apache License, Version 2.0</name>

--- a/xmltordf/pom.xml
+++ b/xmltordf/pom.xml
@@ -4,7 +4,7 @@
     <groupId>no.acando</groupId>
 
     <artifactId>xmltordf</artifactId>
-    <version>1.4.13</version>
+    <version>1.4.14</version>
     <modelVersion>4.0.0</modelVersion>
 
     <name>XmlToRdf</name>

--- a/xmltordf/src/main/java/no/acando/xmltordf/AdvancedSaxHandlerString.java
+++ b/xmltordf/src/main/java/no/acando/xmltordf/AdvancedSaxHandlerString.java
@@ -20,6 +20,7 @@ import org.xml.sax.SAXException;
 
 import java.io.OutputStream;
 import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
@@ -34,7 +35,11 @@ final class AdvancedSaxHandlerString extends AdvancedSaxHandler<String, String> 
     AdvancedSaxHandlerString(OutputStream out, Builder.AdvancedStream builder) {
         super(builder);
 
-        this.out = new PrintStream(out);
+        try {
+            this.out = new PrintStream(out, false, "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            throw new RuntimeException("utf8 encoding could niot be found");
+        }
 
     }
 

--- a/xmltordf/src/main/java/no/acando/xmltordf/FastSaxHandler.java
+++ b/xmltordf/src/main/java/no/acando/xmltordf/FastSaxHandler.java
@@ -22,6 +22,7 @@ import org.xml.sax.SAXException;
 
 import java.io.OutputStream;
 import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
 import java.util.ArrayDeque;
 import java.util.Deque;
 
@@ -46,7 +47,11 @@ public class FastSaxHandler extends org.xml.sax.helpers.DefaultHandler {
     Builder.Fast builder;
 
     public FastSaxHandler(OutputStream out, Builder.Fast builder) {
-        this.out = new UndoableBufferedPrintWriter(new PrintStream(out, false));
+        try {
+            this.out = new UndoableBufferedPrintWriter(new PrintStream(out, false, "UTF-8"));
+        } catch (UnsupportedEncodingException e) {
+            throw new RuntimeException("unsupported encoding");
+        }
         this.builder = builder;
     }
 

--- a/xmltordf/src/test/java/XmlToRdfTest.java
+++ b/xmltordf/src/test/java/XmlToRdfTest.java
@@ -15,6 +15,7 @@ limitations under the License.
  */
 
 import no.acando.xmltordf.*;
+import org.apache.commons.io.FileUtils;
 import org.apache.jena.datatypes.xsd.XSDDatatype;
 import org.apache.jena.graph.NodeFactory;
 import org.apache.jena.rdf.model.Model;
@@ -1351,7 +1352,7 @@ public class XmlToRdfTest {
         Repository repository = build.convertToRepository(new FileInputStream(testFiles.xml));
 
         String rdf = repositoryToString(repository, RDFFormat.JSONLD);
-        Model actualModelSesame = ModelFactory.createDefaultModel().read(new ByteArrayInputStream(rdf.getBytes()), "", RDFLanguages.strLangJSONLD);
+        Model actualModelSesame = ModelFactory.createDefaultModel().read(new ByteArrayInputStream(rdf.getBytes("UTF-8")), "", RDFLanguages.strLangJSONLD);
 
         Model expectedModel = FileManager.get().readModel(ModelFactory.createDefaultModel(), testFiles.expected.getCanonicalPath());
 

